### ISSUE TITLE
Reminder notifications for correspondence games

### ISSUE THREAD

**Issue by joshproehl** [veloce/lichobile#1443]
I often lose the notification that it's my turn in a correspondence game. (Due to network issues, phone reboots, etc.)

Because of this I would like to be able to tell the application to remind me about games that I have which are awaiting my input.
The "You're about to lose against" notification is a good example, and extremely helpful, but on games which have a longer delay I would like to be notified more frequently than just before the game ends.

Ideally I would like to be able to control the frequency. "Notify me once per day", "notify me once every 6 hours" perhaps?

**Comment by jthemphill** [comment]
My app currently has a notification badge telling me about a correspondence game. I’m waiting for my opponent to make a move in that game, and I seem to be unable to clear this notification in the meantime.

**Comment by landry314** [comment]
Yes, warnings don't seem to be working anymore... and I totally agree that multiple warnings would be best. Sometimes a notification gets swiped away while busy or somehow gets lost and then we lose the game. I would like multiple warnings and to be able to set how frequently would be amazing! But even without that, like 8 hours then 4 hours then 1 hour kinda makes sense...
